Lib/idlelib/replace.py

```diff
@@ -10,9 +10,16 @@
 from idlelib.searchbase import SearchDialogBase
 from idlelib import searchengine
 
+
 def replace(text):
-    """Returns a singleton ReplaceDialog instance.The single dialog
-     saves user entries and preferences across instances."""
+    """Create or reuse a singleton ReplaceDialog instance.
+
+    The singleton dialog saves user entries and preferences
+    across instances.
+
+    Args:
```

Comment:
Why don't use ```Parameters```?

Reply:
Mostly consistency.  Other modules in IDLE use `Args:`.  

There are different ways to do list the pararmeters and pep257 isn't specific about which one to use.  There is the java style, Sphinx/reST style, Google style, etc.